cellprofiler/settings.py

```diff
@@ -722,6 +722,25 @@ def set_value_text(self, value_text):
             self.__default = self.str_to_value(value_text)
         except:
             logger.debug("Number set to illegal value: %s" % value_text)
+            
+    def set_min_value(self, minval):
```

Comment:
Would it be possible to use a field?

```Python
class Example:
    self.maxval = None

    self.minval = None
```

If not, would you mind using the property decorator?

```Python
class Example:
    @property
    def min_value(self):
        return self._minval

    @min_value.setter
    def min_value(self, value)
        self._minval = value
```